smac/optimizer/epils.py

```diff
@@ -149,7 +149,27 @@ def run(self):
         """
         self.stats.start_timing()
         try:
-            self.incumbent = self.initial_design.run()
+            initial_design_configs = self.initial_design.select_configurations()
```

Comment:
This is now substantially different than smbo.py. Is it possible to inherit from smbo.py here?